src/controllers/admin.controller.js

```diff
@@ -79,7 +67,7 @@ adminCtrl.deleteStory = (req, res) => {
 adminCtrl.renderEditStoryForm = async(req, res) => {
```

Comment:
add this one after renderStoryList

Reply:
by this, it did refers the function renderEditStoryForm